test/evergreen.yml

```diff
@@ -2659,6 +2677,23 @@ buildvariants:
     - name: ".stress-test-3"
     - name: ".stress-test-4"
 
+- name: performance-tests
+  display_name: Ubuntu 18.04 Performance tests
```

Comment:
This feels to be a very general variant name. I'm not sure if a more specific name e.g. `large scale tests` would make more sense here considering it only includes one perf correctness task right now. 
@sulabhM what's your opinion on this?

Reply:
We could effectively reused the terminology used for the million collection test (see #6681).
`name: large-scale-tests`
For the display name, we could also reuse `~Large scale tests`. What is the `~` though ?

Reply:
Sounds good. The leading `~` is used for those least important or most expensive build variants, which does not seem to apply here.